server/odc-core/src/main/java/com/oceanbase/odc/core/alarm/AlarmEventNames.java

```diff
@@ -56,4 +56,11 @@ private AlarmEventNames() {}
 
     public static final String SCHEDULING_FAILED = "SCHEDULING_FAILED";
 
+    /**
+     * TASK
+     */
+    public static final String TASK_CANCELED_FAILED = "TASK_CANCELED_FAILED";
```

Comment:
I think adding a task failure alarm would help developers understand the task success rate and facilitate code optimization.